README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
 </settings>
 ```
 
-### More information
+#### More information
+
 - https://central.sonatype.org/pages/ossrh-guide.html
+
+## Configuration options
+
+The `UnleashConfig` class exposes a set of builder methods to configure your Unleash client. The available options are listed below with a description of what they do. For the full signatures, take a look at the [`UnleashConfig` class definition](src/main/java/io/getunleash/util/UnleashConfig.java).
+
+| Method name                                | Description                                                                                                                                                         | Required | Default value                        |
+|--------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------|
+| `appName`                                  | The name of the application as shown in the Unleash UI. Registered applications are listed on the Applications page.                                                | Yes      | `null`                               |
+| `backupFile`                               | The path to the file where [local backups](#local-backup) get stored.                                                                                               | No       | `"java.io.tmpdir/unleash-repo.json"` |
+| `customHttpHeader`                         | Add a [custom HTTP header](#custom-http-headers) to the list of HTTP headers that will the client sends to the Unleash API. Each method call will add a new header. | No       | N/A                                  |
```

Comment:
Presumably, this isn't a required setting _per se_. However, this seems to be the only way to set the authentication token, and the client won't work without that, right?

Reply:
I can see a use case here where someone turns off Auth on the Unleash server side. A few of the SDKs have this as an optional property. Might be worth a warning that for the overwhelming majority of cases this is required to make the system work though!